daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/InterfaceViewSpec.scala

```diff
@@ -59,59 +58,17 @@ class InterfaceViewSpec extends AnyWordSpec with Matchers with EitherValues with
       .computeInterfaceView(templateId, argument, interfaceId)
       .consume(_ => None, lookupPackage, _ => None)
   private val t1 = id("T1")
-  private val t2 = id("T2")
-  private val t3 = id("T3")
-  private val t4 = id("T4")
   private val i = id("I")
   private val iNoView = id("INoView")
 
   "interface view" should {
 
-    "return result of view method when it succeds" in {
+    // TODO: Disabled following tests for https://github.com/digital-asset/daml/pull/14456,
```

Comment:
Please try to keep `computeInterfaceViw` working in some form at all points so we don’t end up blocking the participant team on their work.

Reply:
I've opted to add _view method back to InterfaceViews.daml, that should keep the test passing until we fully remove the `_view` functionality from the engine in the next PR